plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -584,6 +584,77 @@ public void testAddColumnConcurrently()
         throw new SkipException("TODO");
     }
 
+    @Test
```

Comment:
Please also add a test with nested data. Nesting random stuff from tpch should be enough.

Reply:
And with arrays if possible - for a specific index